entry.go

```diff
@@ -108,16 +113,13 @@ func (te *TunaEntry) Start() {
 			if err != nil {
 				return
 			}
-			stream, err := session.OpenStream()
-			if err != nil {
-				return
-			}
-			stream.Close()
+
 			for {
 				_, err = session.AcceptStream()
 				if err != nil {
 					log.Println("Close connection:", err)
-					return
+					time.Sleep(time.Second * 1)
```

Comment:
When a TCP accept returns error, you should return instead of continue because it means the connection is closed or cannot accept connection anymore.